packages/grafana-ui/src/options/models.gen.ts

```diff
@@ -16,14 +16,14 @@ export interface VizTextDisplayOptions {
  * @public
  */
 export interface OptionsWithLegend {
-  legend: VizLegendOptions;
+  legend?: VizLegendOptions;
```

Comment:
We don't want to change this to optional

Reply:
Yeah, i was concerned about that. But validation fails here if it's required and has no default - there's devenv dashboards that are missing those values.

Reply:
and unfortunately these were some of the least helpful error messages - for some reason they DIDN'T come with the usual line numbers that CUE dumps out 😠 - something to def work on